optimizely/event/event_processor.py

```diff
@@ -222,7 +222,7 @@ def flush(self):
 
     def _flush_queue(self):
         """ Flushes event_queue by dispatching events. """
-
+        self.logger.debug('Flushing the queue.')
```

Comment:
Actually it will be better to look up if events are there and say nothing to flush if no events and then say flushing the queue if there are events. Also, perhaps number of user events flushed will be useful.